src/components/cookie-banner/index.md.njk

```diff
@@ -29,95 +40,112 @@ Check with your organisation's privacy expert to see how data protection legisla
+#### When the user has accepted cookies
+
+Unhide a replacement message confirming that the user has either accepted or rejected cookies by removing the hidden attribute.
 
 {{ example({group: "components", item: "cookie-banner", example: "accepted", html: true, nunjucks: true, open: false}) }}
 
+#### When the user has rejected cookies 
+
 {{ example({group: "components", item: "cookie-banner", example: "rejected", html: true, nunjucks: true, open: false}) }}
 
-Write your own code to:
+## What to cover in your cookie banner
 
-- update the cookies you’re setting on that user’s device - including removing any types of non-essential cookies the service has set previously, but that the user is no longer consenting to
-- set a cookie remembering that the user has seen the cookie banner and updated their cookie settings - this cookie should expire after 1 year
-- remove the cookie banner if the user clicks on the ‘Hide this message’ button
-- prevent the cookie banner being shown when the user moves on to a new page
+Include the name of the service in the banner heading to help users understand that the cookies you’re talking about are different from the ones set by the main GOV.UK platform.
 
-## If the user’s device is not running JavaScript
+You’ll need to change the example cookie banner text if your service:
 
-If the user’s device is not running JavaScript, display a version of the cookie banner which tells them what to do in order to set or change their cookie preferences. Do not set any non-essential cookies, because the user has not consented to them.
+- allows third parties to set cookies (tell the user that both your organisation and other organisations will be setting cookies)
+- uses cookies for reasons other than collecting analytics information or remembering the user’s settings
 
-{{ example({group: "components", item: "cookie-banner", example: "no-js", html: true, nunjucks: true, open: false}) }}
+Keep the text as short as possible while making sure it’s an accurate description of how you use cookies. For example, if you use more than one ‘functional’ cookie and there’s not enough space to mention what each of them does, you could ask for permission to set cookies so ‘you can use as many of the service’s features as possible’.
 
-If your service only uses essential cookies, display a different version of the cookie banner to users whose devices are not running JavaScript.
+[See the cookies page for more advice on writing about cookies(/patterns/cookies-page/)].
 
-{{ example({group: "components", item: "cookie-banner", example: "essential-no-js", html: true, nunjucks: true, open: false}) }}
+### If you’re using essential cookies and analytics cookies
 
-## Getting consent from users who are not running JavaScript
+You can use this example text for a service which sets essential and analytics cookies - set by your organisation to collect analytics information about how well the service is performing.
```

Comment:
Got a bit stumped by this sentence, wonder if it could be broken up to make it clearer...

Reply:
```suggestion
You can use this example text for a service which sets essential and analytics cookies. Analytics cookies are those set by your organisation to collect information about how people are using your digital service
```